sympy/printing/latex.py

```diff
@@ -1497,7 +1497,7 @@ def _print_MatMul(self, expr):
         from sympy import Add, MatAdd, HadamardProduct, MatMul, Mul
 
         def parens(x):
-            if isinstance(x, (Add, MatAdd, HadamardProduct)):
+            if isinstance(x, (Add, MatAdd, HadamardProduct)) or isinstance(x, MatMul) and x.args[0] < 0:
```

Comment:
what about using `return self.parenthesize(x, precedence_traditional(expr))` ?

The function `parens` could be removed.

Reply:
@Upabjojr That seems to alter other behaviour
`l._print_MatMul(2*x*A)` before was `'2 x A'` but now gives `'\\left(2 x\\right) A'`
This is the test on line 1228

Reply:
Sorry, missing a parameter, `self.parenthesize(x, precedence_traditional(expr), strict=True)`

Reply:
Then it doesn't add brackets altogether, not even for `latex((x * y).subs(y, -y))`

Reply:
Perhaps the precedence rules need to be modified. I agree it would be better if this could use `parenthesize` rather than this special function.